core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaRespawnTranslator.java

```diff
@@ -100,6 +100,7 @@ public void translate(GeyserSession session, ClientboundRespawnPacket packet) {
                 DimensionUtils.switchDimension(session, fakeDim);
             }
             session.setWorldName(spawnInfo.getWorldName());
+            session.setWorldTicks(0);
```

Comment:
question - shouldn't this also reset the `ticks` on the session too? Or is there a specific reason for those not being reset

Reply:
My intention is for `ticks` to never go backwards and be used for comparing time between two events, similar to `System.currentTimeMillis()`. `worldTicks` is reset here to match java client behavior